### PR TITLE
Fix chat example

### PR DESCRIPTION
**Pull request by raulk**
* display actual port in instructions when starting server side.
* make debug mode false by default, as otherwise the same peer ID is generated every time because the source of randomness is constant.
* fix some typos and comments.

### CODE REVIEW

File: chat/chat.go [hunk 1]
```diff
@@ -178,31 +179,44 @@ func main() {
 
 	if *dest == "" {
 		// Set a function as stream handler.
-		// This function  is called when a peer initiate a connection and starts a stream with this peer.
-		// Only applicable on the receiving side.
+		// This function is called when a peer connects, and starts a stream with this protocol.
+		// Only applies on the receiving side.
 		host.SetStreamHandler("/chat/1.0.0", handleStream)
 
-		fmt.Printf("Run './chat -d /ip4/127.0.0.1/tcp/%d/ipfs/%s' on another console.\n You can replace 127.0.0.1 with public IP as well.\n", *sourcePort, host.ID().Pretty())
```
Comment (upperwal) [inline chat/chat.go]: Better to replace `ipfs` with `p2p` and use `P_P2P` in `addAddrToPeerstore`

Comment (raulk) [reply chat/chat.go]: Good point!

**Comment by upperwal** [comment]
I think, removing `addAddrToPeerstore ` and using `pstore.InfoFromP2pAddr` instead of it would be a good idea. It will reduce the complexity of the example.

**Comment by raulk** [comment]
@upperwal makes sense! updated.

**Comment by raulk** [comment]
@bigs could you have a quick look, please? super simple.

**Pull request merged**